browser/ui/views/location_bar/onion_location_view.cc

```diff
@@ -71,8 +72,6 @@ class OnionLocationButtonView : public views::LabelButton {
                     brave_l10n::GetLocalizedResourceUTF16String(
                         IDS_LOCATION_BAR_OPEN_IN_TOR)),
         profile_(profile) {
-    SetTooltipText(brave_l10n::GetLocalizedResourceUTF16String(
```

Comment:
For some reason, when I tested this branch locally on my machine (Linux), the tooltip in normal windows and in Tor windows showed: "Open in Tor <URL>" instead of just "<URL>".

Reply:
Very strange :/ Looks like it is impossible, I guess you didn't pull the last change.

Reply:
You're right, it looks like I somehow had an old version of your branch. I pulled again and it works as expected.